Library/Formula/ipmitool.rb

```diff
@@ -2,18 +2,17 @@
 
 class Ipmitool < Formula
   homepage 'http://ipmitool.sourceforge.net/'
-  url 'https://downloads.sourceforge.net/project/ipmitool/ipmitool/1.8.13/ipmitool-1.8.13.tar.bz2'
-  sha1 '22254a2b814c8cd323866a4dd835e390521c1dfa'
+  url 'https://downloads.sourceforge.net/project/ipmitool/ipmitool/1.8.15/ipmitool-1.8.15.tar.bz2'
+  sha1 '2c9c5d7c5a285586df508ad933577f275684353a'
+
+  depends_on "openssl"
 
   def install
     # tracking upstream: http://sourceforge.net/p/ipmitool/feature-requests/47/
     # fix build errors w/ clang
     inreplace 'include/ipmitool/ipmi_user.h', 'HAVE_PRAGMA_PACK', 'DISABLE_PRAGMA_PACK'
-    # undefined non-posix symbols
-    inreplace 'src/plugins/serial/serial_basic.c', 'IUCLC', '0'
-    inreplace 'src/plugins/serial/serial_basic.c', 'TCFLSH', '0'
-    inreplace 'src/plugins/serial/serial_terminal.c', 'IUCLC', '0'
-    inreplace 'src/plugins/serial/serial_terminal.c', 'TCFLSH', '0'
+    # s6_addr16 is not available in Mac OS X
```

Comment:
Can you please submit this inreplace as a patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!